### PR TITLE
Modified AzureUSGov Endpoint to .US

### PR DESCRIPTION
**Pull request by bernie-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [X] Each command and parameter has a meaningful description.
- [X] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by derekbekoe** [comment]
Please modify HISTORY.rst with the change.

**Comment by bernie-msft** [comment]
Updated HISTORY.rst, it should show up now.

File: src/azure-cli-core/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.0.22
+++++++
+* Modified the AZURE_US_GOV_CLOUD's AAD authority endpoint from login.microsoftonline.com
```
Comment (yugangw-msft) [inline src/azure-cli-core/HISTORY.rst]: Please make it a single line
Also, please update the [setup.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L18) under the same directory to have the same version

Comment (bernie-msft) [reply src/azure-cli-core/HISTORY.rst]: Ok thanks, I've reduced the HISTORY.RST change to a single line and updated the version in setup.py. Let me know what else you guys need. Thx!

**Comment by bernie-msft** [comment]
I don't understand what's being asked. The CI build appeared to fail with nothing to do with my commits.

**Comment by derekbekoe** [comment]
@bernie-msft We released a new update to the CLI yesterday and so the version numbers for azure-cli and azure-cli-core were changed. The PR for that was merged yesterday ([see this](https://github.com/Azure/azure-cli/pull/4882/files)).
Your PR doesn't include this change and so CI fails. On your local machine pull from the dev branch, and merge the latest of that branch to your azure-cli-govSTSendpointupdate branch. I suspect there will be a conflict (as the version no. change was also made in that other PR) so resolve that conflict then push azure-cli-govSTSendpointupdate. This will update to allow CI to run again. It should be green after that then we can merge.

**Comment by bernie-msft** [comment]
I see, thanks. So, should I updated to version 2.0.23 in my next commit?

**Comment by derekbekoe** [comment]
In history.rst, the heading `2.0.22` will already exist so you can remove '* Minor fixes' and replace with your change message.
You don't need to change the version in setup.py.

**Comment by bernie-msft** [comment]
Ok thanks, I believe I've made the appropriate change.

**Comment by derekbekoe** [comment]
There was an error with HISTORY.rst. "Bullet list ends without a blank line; unexpected unindent.".
Add a new line after the bullet list.

**Comment by bernie-msft** [comment]
Apologies, this is corrected.

**Pull request merged**